scripts/schemas.py

```diff
@@ -34,32 +34,19 @@ def create_csv(fields, file):
                 schema_writer.writerow([field["name"], field["type"], field["level"], field["example"]])
 
 
-def create_markdown(fields, file):
```

Comment:
Trying to understand how this removal is related to this PR? Just a left over?

Reply:
Just old dead code. As I mentioned above ^ I'm just trying to lump more changes together, since we have a lot to do.

But I agree cleanups should normally be their own PR as well.